image_client/nightly_sync.py

```diff
@@ -93,13 +103,17 @@ def main():
             ich_importer_config.SPECIFY_DATABASE_PORT,
             ich_importer_config.SPECIFY_DATABASE_HOST,
             ich_importer_config.SPECIFY_DATABASE)
-    do_import(collection_name,specify_db_connection)
+    do_sync(collection_name, specify_db_connection)
 
 
 if __name__ == '__main__':
```

Comment:
failed merge as part of commit. I suspect this is again my change showing up ehre somehow... but fix it because may as well get it sorted now.